docs/HowToGuides/GettingStarted.md

```diff
@@ -190,6 +190,9 @@ toolchain as a one-off, there are a couple of differences:
 > LLDB currently requires at least `swig-1.3.40` but will successfully build
 > with version 2 shipped with Ubuntu.
 
+> **Note**  
+> When building on Linux, make sure to have at least 8 GB of RAM when using flag `--release-debuginfo` and 32 GB of RAM when using flag `--debug-swift` and 150 GB of disk space available.
```

Comment:
and 150 -> , plus 150

Too many `and`s otherwise

Reply:
Adjusted 🤝